app/views/events/index.haml

```diff
@@ -0,0 +1,48 @@
+%h2 My events
+- if current_user.events.any?
+  %ul.list-group
+    - @user_events.each do |event|
+      %li.list-group-item
+        .row
+          .col-xs-3
+            =  image_tag(event.picture, class: "img-responsive")
+          .col-xs-9
+            .media-body
+              %h3.media-heading
+                = event.title
+              .row
+                .col-xs-12.col-md-6
+                  .row.no-gutter
+                    .col-xs-3.text-right.truncate
```

Comment:
you used `truncate` class on tags with fixed width of characters inside like `Category:` I don't see the point. Am I missing something?